packages/warehouses/src/warehouseClients/BigqueryWarehouseClient.ts

```diff
@@ -530,7 +530,7 @@ export class BigqueryWarehouseClient extends WarehouseBaseClient<CreateBigqueryC
             )}`,
         );
         return new WarehouseQueryError(
-            `Bigquery warehouse error: ${error?.reason}`,
+            `Bigquery warehouse error: ${error?.reason} - ${error?.message}`,
```

Comment:
we intentionally didn't include `error?.message` in case that returned some private info. Could you remove it ? 
This info is already returned on a console.error, so it can be reviewed on the backend logs